### ISSUE TITLE
changes to BS tabs styles

### ISSUE THREAD

**Issue by likello** [osu-wams/madrone#230]
```
.block-views {
    .nav-tabs {
        border-width: 0;
        border-color: $osuorange;
        flex-wrap: nowrap;
        overflow-x: auto;
        scroll-snap-type: x mandatory;
        -ms-overflow-style: none; // IE and Edge
        scrollbar-width: none; // Firefox

        &::-webkit-scrollbar {
            display: none; //Chrome
        }

        @media (min-width:992px) {
            border-width: .15em;
            overflow: visible;
            justify-content: center;
            margin-inline: 2rem;
        }

        @media (min-width:1400px) {
            margin-inline: 4rem;
        }

        .nav-item {
            margin-inline-end: 0.25rem;
            scroll-snap-align: end;
            width: max-content;
        }

        .nav-link {
            border: 0;
            border-top-right-radius: 0.25rem;
            border-top-left-radius: 0.25rem;
            transition: background-color 0.2s ease-out;
            width: max-content;

            &.active {
                border-bottom: .15em solid $osuorange;
                margin-bottom: 0;
            }

            &:hover {
                background-color: $osuorange;
                color: $bucktoothwhite;
            }

            @media (min-width: 990px) {

                &.active {
                    border-block-start: .15em solid $osuorange;
                    border-inline: .15em solid $osuorange;
                    border-bottom: 0;
                    margin-bottom: -.14em;
                }
            }

            &:focus-visible {
                box-shadow: none;
                outline: 2px solid $paddletailblack;
            }
        }
    }
}
```

**Comment by likello** [comment]
updated code

**Comment by emerham** [comment]
Fixed in #236
